test/e2e/test_cluster_configs/custom_hyperkube.json

```diff
@@ -0,0 +1,47 @@
+{
+    "options": {
+        "allowedOrchestratorVersions": [
+            "1.16"
+        ]
+    },
+    "apiModel": {
+        "apiVersion": "vlabs",
+        "properties": {
+            "orchestratorProfile": {
+                "orchestratorType": "Kubernetes",
+                "orchestratorRelease": "1.16",
```

Comment:
nit: this is actually superfluous because the Jenkinsfile will interpret the `allowedOrchestratorVersions` array before generating api models, and inject the `orchestratorRelease` value that correlates with the value in the array.